.travis.yml

```diff
@@ -41,7 +40,7 @@ before_script:
 script:
     - mkdir -p build
     - cd build
-    - cmake ..
+    - cmake -D CMAKE_PREFIX_PATH:PATH=${DEPS_DIR}/protobuf/install ..
```

Comment:
The -D command-line flag vs. environment variable is mostly just our default practice to avoid reliance on environment variables which are passed through to other processes. The reason we don't append the PATH is that CMAKE_PREFIX_PATH is just a set of additional paths to look for packages, which defaults to empty. cmake will continue to use the PATH environment variable normally in any case.